### DISCUSSION TITLE
Lab 27 Task 3

### DISCUSSION THREAD

**Discussion by achou05** [hack4impact-calpoly/starter-pack#11] [Q&A] [answered]
I was trying to execute git push -u origin main in my terminal and I keep receiving Authentification fail, does anyone have any ideas? 

<img width="1406" alt="fail" src="https://github.com/hack4impact-calpoly/starter-pack/assets/123144414/d641e5e9-14ce-4ba8-ae3e-38b1957d5b85">

**Comment by NoahMoscovici** [comment, marked as answer]
Github doesn't allow password authentication anymore, you can probably follow [these instructions](https://stackoverflow.com/a/68781050) to set up an alternative way to sign in to Github

**Reply by achou05** [reply]
Thank you very much, my guy. Appreciate the reply, I literally just tried everything and it worked somehow, so ITS LIT. Again thank you very much sir 😄. If anyone else is having trouble with this make a comment and I'll see what I can do to help!
